### ISSUE TITLE
the precision of SiamRPN-VGG

### ISSUE THREAD

**Issue by zhanght021** [leeyeehoo/SiamRPN-VGG#5]
Hi, I am a little confused about the precision, in the paper "SiamVGG", the precision of SiamVGG is 
 a little better than SiameseRPN, but here with the module RPN, the precision of SiamRPN-VGG is a little worse than SiameseRPN, the EAO of VOT2018 is only 0.21, I am confused,请求大神指教

**Comment by fAnchao1** [comment]
@zhanght021 你用他这个代码可以训练嘛

**Comment by zhanght021** [comment]
@fAnchao1 没呢，我就看到了精度问一下，那个HelloRicky123不是已经搞定代码了吗，他今天又更新了，在OTB100上的精度和原文几乎一致了，并且他还给出了他训练好的模型，你可以去看一下

**Comment by fAnchao1** [comment]
@zhanght021 哦好的谢谢啦

**Comment by fAnchao1** [comment]
@zhanght021 我没跑通啊，那个人的代码真的不友好

**Comment by leeyeehoo** [comment]
对 用他的。。。 我这个估计是网络架构问题， 我用他的改的我的vgg还是不行 他最后能clsloss收到0.03左右vgg只能到0.09

**Comment by zhanght021** [comment]
@leeyeehoo 有一个问题，我看你在image.py中， iou>0.4就认为是正样本，这个值会不会太小了，我看SiameseRPN中用的是0.6，然后就是在loss.py中pos_index = np.random.choice(np.where(target[batch_id].cpu() == 1)[0], num_pos)，如果正样本数不够num_pos，那不就会重复其中的某几个样本了吗

**Comment by leeyeehoo** [comment]
弄反了吧。。。>0.6正样本数量才会更少O__O "…

**Comment by zhanght021** [comment]
@leeyeehoo 对，那这个阈值设置的太小的话，正样本是变多，但是这些正样本中岂不是很多和真实目标比较不接近，阈值设置的大一点不是更加的合理，

**Comment by leeyeehoo** [comment]
我在实验中两个都试过， 0.4附近会更好。不过主要应该是网络结构问题。我怎么train都不大行。

**Comment by haishibei** [comment]
@leeyeehoo 请问SiamVGG和SiamRPN的text_txt 文件可以分享一下吗？
